pdfminer/utils.py

```diff
@@ -293,6 +293,21 @@ def matrix2str(m):
     return '[%.2f,%.2f,%.2f,%.2f, (%.2f,%.2f)]' % (a, b, c, d, e, f)
 
 
+
+def mean(it):
+    sum=0
+    count=0
+    for el in it:
+        sum+=el
+        count+=1
+    return sum / count
+
+def itemz(it):
```

Comment:
I find the name of this function perculiar. Maybe rename it to `key_values()`?

Reply:
or maybe `universal_keys`